### ISSUE TITLE
PageBreak Filter doesn't work for org files

### ISSUE THREAD

**Issue by mak1A4** [pandoc/lua-filters#159]
It seems that the pagebreak filter doesn't work for org files, or I'm doing something wrong but when I use the filter with markdown as source it works just fine. I'm using pandoc version 2.11.4 installed with deb package.

**Comment by tarleb** [comment]
You are right, just inserting `\newpage` doesn't work for org mode. We may want to support that.

The recommended method for org mode is to use a "form feed" character surrounded by blank lines. You can insert it with <kbd>C-q C-l</kbd>.

The alternative is to use

``` org
#+LaTeX: \newpage
```

**Comment by mak1A4** [comment]
Thanks that worked perfectly, greatly appriacted. Maybe it would be worth to add this to the readme, I couldn't figure this one out myself just by googeling.
